gamemode/cl_init.lua

```diff
@@ -247,8 +247,15 @@ net.Receive("Horde_SyncClasses", function ()
     HORDE.classes = net.ReadTable()
 end)
 
-hook.Add("HUDShouldDraw", "RemoveRetardRedScreen", function(name) 
+hook.Add("HUDShouldDraw", "RemoveRetardRedScreen", function(name)
     if (name == "CHudDamageIndicator") then
        return false
     end
+end)
+
+hook.Add("PlayerBindPress", "HelpfulBinds", function(ply, bind, pressed)

```

Comment:
Would you be able to add a `Horde_` prefix in front of the hook? Trying to make things consistent. I noticed that I am missing that for my own function as well. You can change that if you want, or I will fix it myself.

Reply:
Done